src/System.Drawing.Primitives/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1 @@
+user32.dll!GetSysColor
```

Comment:
@hughbe thanks for checking. No it's not. As you see in src\System.Drawing.Primitives\dir.,props `    <IsUAP>true</IsUAP>` which means it needs to work in UWP apps. Therefore PINvokes must all go through the WACK (the legal list for UWP) and GetSysColor is getting flagged because it isn't there. Maybe it should be, but it's too late to ask now.
This may be why it has its own copy of KnownColors.